### PR TITLE
Add plopp to installation environments

### PR DESCRIPTION
**Pull request by nvaytet**

### CODE REVIEW

File: docs/environments/ess-mantid.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ dependencies:
   - tifffile
   - jupyterlab
   - jupyterlab_widgets
+  - plopp
   - pythreejs
+  - scipp<23.08.0  # TODO: remove version pin when mantid and scipp have compatible versions of TBB
```
Comment (jl-wynen) [inline docs/environments/ess-mantid.yml]: Is this needed? Can't conda resolve to a compatible version automatically?

Comment (nvaytet) [reply docs/environments/ess-mantid.yml]: Actually it does. I'm not sure why I thought we needed this. Wojciech said he ran into the issue. I'll remove.

**Review by jl-wynen — APPROVED** [review]

File: docs/environments/ess-mantid.yml (hunk 1 above)
Comment (jl-wynen) [reply docs/environments/ess-mantid.yml]: Did he make an env from scratch? It could be that it fails to resolve when you install into an existing env.

**Comment by nvaytet** [comment]
Flaky macos docs build failed. Other builds fine. Merging.

**Pull request merged**